### PR TITLE
LES-453 add designe for admin part category page + link in Side Bar only for admin

### PR DESCRIPTION
**Pull request by NatashaMikhaltsova**

### CODE REVIEW

File: src/components/body/content/categories/actions/categoryImage.scss [hunk 1]
```diff
@@ -1,83 +1,94 @@
...
+  background: $blueLightColor;
+  color: $white;
+  margin: 0;
+  width: 188px;
+  height: 32px;
 
-.category-upload {
-  margin: 12px 64px 50px 64px;
+  &:disabled {
+    background: #FFFFFF;
```
Comment (Kurilo-Pavel) [inline src/components/body/content/categories/actions/categoryImage.scss:29]: use constabt $white

File: src/components/body/content/categories/actions/categoryImage.scss [hunk 2]
```diff
@@ -1,83 +1,94 @@
...
-  width: 100%;
-  margin-top: 4px;
-  background: #FBFBFF;
-  height: 216px;
-  box-sizing: border-box;
-}
+  justify-content: center;
+  border: 1px solid $blueLightColor;
+  border-radius: 28px;
+  font-family: "ApercuPro";
```
Comment (Kurilo-Pavel) [inline src/components/body/content/categories/actions/categoryImage.scss:18]: use constant $font

File: src/components/body/content/categories/actions/categoryImage.scss [hunk 3]
```diff
@@ -1,83 +1,94 @@
...
-  height: 0;
-  visibility: hidden;
-  opacity: 0;
-  padding: 0;
+.category-image-button-delete {
+  align-items: center;
+  justify-content: center;
+  border: 1px solid $warningColor;
+  border-radius: 28px;
+  font-family: "ApercuPro";
```
Comment (Kurilo-Pavel) [inline src/components/body/content/categories/actions/categoryImage.scss:63]: use constant $font

File: src/components/body/content/categories/actions/categoryImage.scss [hunk 4]
```diff
@@ -1,83 +1,94 @@
...
 
-.button-select {
-  display: flex;
+.category-image-button-change {
   align-items: center;
+  justify-content: center;
   border: 1px solid $blueLightColor;
   border-radius: 28px;
-  font-family: "ApercuPro", sans-serif;
+  font-family: "ApercuPro";
```
Comment (Kurilo-Pavel) [inline src/components/body/content/categories/actions/categoryImage.scss:79]: use constant $font

File: src/components/button.scss [hunk 5]
```diff
@@ -50,13 +50,14 @@
   color: $blueLightColor;
   padding: 6px 36px;
   background: $white;
+  justify-content: center;
 
   &:disabled {
-    background: $white;
+    background: #F3F3F3;
```
Comment (Kurilo-Pavel) [inline src/components/button.scss:56]: use constant or create it

**Review by Kurilo-Pavel — APPROVED** [review]

**Review by Terapool — APPROVED** [review]

**Review by Terapool — APPROVED** [review]

**Pull request merged**